crates/bitwarden-vault/src/folder_client.rs

```diff
@@ -35,4 +70,56 @@ impl FoldersClient {
         let views = key_store.decrypt_list(&folders)?;
         Ok(views)
     }
+
+    /// Create a new folder and save it to the server.
+    pub async fn create(
+        &self,
+        request: FolderAddEditRequest,
```

Comment:
We could probably implement `Encryptable` directly to `FolderAddEditRequest` which would encrypt into a `FolderRequestModel`?